tests/TestPsci/CommandTest.hs

```diff
@@ -34,7 +41,14 @@ commandTests = context "commandTests" $ do
     length ms' `equalsTo` 3
 
   specPSCi ":complete" $ do
-    ":complete ma" `prints` []
+    ":complete ma" `prints` ""
     ":complete Data.Functor.ma" `prints` (unlines (map ("Data.Functor." ++ ) ["map", "mapFlipped"]))
     run "import Data.Functor"
     ":complete ma" `prints` (unlines ["map", "mapFlipped"])
+
+  specPSCi ":browse" $ do
+    liftIO $ setSample ".updated"
```

Comment:
If we're really going to copy files around then we should put things back after we're done, please, using `bracket` to handle exceptions properly.

Reply:
Wasn't aware of this - thanks for the heads up!